include/zcbor_common.h

```diff
@@ -190,12 +190,11 @@ bool zcbor_union_end_code(zcbor_state_t *state);
 /** Initialize a state with backups.
  *  One of the states in the array is used as a struct zcbor_state_constant object.
  *  This means that you get a state with (n_states - 2) backups.
- *  It also means that (n_states = 2) is an invalid input, which is handled as
- *  if (n_states = 1).
+ *  The constant state is mandatory so n_states must be at least 2.
```

Comment:
Should we document when it will return `true` or `false`?

Reply:
Good idea, done.